src/components/MovieList/MovieList.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./MovieList.module.css";
+import React from "react";
+import {MovieDetailsData} from "../../types/MovieDetailsData";
+import MovieTile from "../MovieTile/MovieTile";
+import {DialogType} from "../../constants/DialogType";
+
+interface MovieListProps {
+    movieList: MovieDetailsData[];
+    handleTileClick: (movie: MovieDetailsData) => void;
+    handleEditMovie: (dialogType: DialogType, movie: MovieDetailsData) => void;
+    handleDeleteMovie: (dialogType: DialogType, movie: MovieDetailsData) => void;
+}
+
+const MovieList: React.FC<MovieListProps> = ({movieList, handleTileClick, handleEditMovie, handleDeleteMovie}) => {
+    const handleEdit = (movie: MovieDetailsData)=> {
+        handleEditMovie(DialogType.EditMovie, movie)
+    }
+
+    const handleDelete = (movie: MovieDetailsData)=> {
+        handleDeleteMovie(DialogType.DeleteMovie, movie)
+    }
+
+    return (
+        <div data-testid="movie-list"
+             className={styles.movieList}>
+            {movieList.map((movie) => (
+                <MovieTile key={movie.id}
+                       movieDetails={movie}
+                       onClick={handleTileClick}
+                       handleEdit={() => handleEdit(movie)}
+                       handleDelete={() => handleDelete(movie)}
+                />
+            ))}
+        </div>)
+}
+
```

Comment:
you have different offsets. 
![image](https://github.com/user-attachments/assets/1ee4c3b3-e0cb-4ed2-be9c-53bd096f564b)

would be better to use eslint or prettier, and turn on run on  save:
![image](https://github.com/user-attachments/assets/4bfe8773-2776-447d-a93e-90f721d0653d)